UQ-eLIPSE/crucible-components
# Story/vethub 38

Added real dataset.

Modified tests according.

To test:
- Run `yarn test`
- Go to https://localhost:5173/ and attempt the questions, see any issues that may be obvious.
